### DISCUSSION TITLE
I want to run multiple PlantModel simultaneously

### DISCUSSION THREAD

**Discussion by yuntoutou** [openTCS/opentcs#118] [Q&A]
I want to run multiple PlantModel simultaneously. How should I modify the source code? Can you give me some ideas

**Comment by swltr** [comment]
@yuntoutou openTCS is designed to work with exactly one plant model. (It's OK to have multiple separate driving courses in the same plant model, though, which I assume is what you actually want.)

**Reply by yuntoutou** [reply]
We have many factories, each with different AGV systems and different points/paths. I want to run multiple factories simultaneously

**Reply by swltr** [reply]
@yuntoutou As I wrote: you can have multiple separate driving courses in the same plant model, which would allow you to do that.

However, for separate factories, I would recommend running separate kernels, one for each factory. This way, modifying the driving course of one factory will not disrupt processes in the other factories.

**Reply by SHMay100G** [reply]
How to implement it?

**Reply by swltr** [reply]
It is unclear to me what you mean. Implement what?

**Reply by freekevinCode** [reply]
> How to implement it?

Deploying multiple kernels， are you ok？
